src/util/PlacesToNodeMapper.cpp

```diff
@@ -105,4 +105,34 @@ std::vector<int> PlacesToNodeMapper::getFileTransferServicePort(long placeId) {
     }
 
     return portList;
+}
+
+
+int PlacesToNodeMapper::countPortRanges(int totalNumberOfWorkers) {
```

Comment:
IMO we should change this return type to void. Because we do not need to know the number of ports counted.